source/collaborate/install-ios-app.rst

```diff
@@ -8,12 +8,16 @@ Take Mattermost wherever you go by installing the Mattermost mobile app on your
 
 1. On your device, visit the App Store. 
 2. Search for "Mattermost" and select **GET** to download the app.
-3. Enter your Apple ID Password or use Face or Touch ID to proceed with the install. You may also be asked to provide an Apple Verification Code received through text/phone call, login notification, or **Settings** on your trusted device.
-4. Open Mattermost and follow the steps below to log in:
+3. Enter your Apple ID Password or use Face or Touch ID to proceed with the installation. You may also be asked to provide an Apple Verification Code received through text/phone call, login notification, or **Settings** on your trusted device.
+4. Open the Mattermost app and click on **Sign in** on the welcome screen which is below the **Next** button.
+5. On the next screen, click on **Allow** when prompted to approve notifications for the Mattermost app.
+6. Now follow the steps below to log in:
 
-   a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. Select **Connect** to continue. See the `access your workspace </collaborate/access-your-workspace.html>`__ documentation for additional details. 
-   b. Enter your credentials as specified by your Mattermost system admin. Select **Log in**.
+   a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. 
+   b. **Display Name:** This is a name for your server so that you can identify it in case you have multiple servers set up in your app. Click on **Connect** to continue.
```

Comment:
```suggestion
   b. **Display Name:** This is a name for your server so that you can identify it in case you have multiple servers set up in your app. Tap **Connect** to continue.
```